### DISCUSSION TITLE
Modifying the "No Decoration" theme

### DISCUSSION THREAD

**Discussion by javathehunt** [piroor/treestyletab#2871] [Q&A]
Hi - Not sure its  appropriate to post this here but thought i  woud try. In the options section for TreeStyleTabs at the top under : Appearance / Theme / No Decoration (Use extra style rules for sidebar content) - What does this mean and how can I customize this  setting? Selecting it provides for a more condensed view I actually prefer but if there is a way to tweak it a bit that would be helpful. I see the section down below called - Extra style rules for contents provided by tree style tab - I dont understand how to use it. I see the window with drop down menu for Theme and that does nothing for sidebar view. Basically if I can just change the spacing in the sidebar slightly that would be a big help. Can someone give me an idea how to customize this view and where to start? Thanks.

![12](https://user-images.githubusercontent.com/76752435/115970163-489a0a00-a530-11eb-9615-fc5c6895c7f6.png)

![11](https://user-images.githubusercontent.com/76752435/115970161-459f1980-a530-11eb-98b6-ba115b6ee6ab.png)

**Comment by piroor** [comment]
TST's UI is built on generic HTML and CSS. This means that you can define any favorite visual design like https://github.com/piroor/treestyletab/wiki/Metal-theme with generic knowledge about building web pages. In other words, if you don't know it, [you need to study it at first](https://developer.mozilla.org/en-US/docs/Web/CSS).

And you'll need to [inspect TST's sidebar page, to write effective CSS](https://github.com/piroor/treestyletab/wiki/How-to-inspect-tree-of-tabs#how-to-inspect-the-sidebar). Reading CSS of TST's built-in theme may help to understand how to customize:

https://github.com/piroor/treestyletab/tree/trunk/webextensions/sidebar/styles/photon
https://github.com/piroor/treestyletab/tree/trunk/webextensions/sidebar/styles/proton
https://github.com/piroor/treestyletab/tree/trunk/webextensions/sidebar/styles/sidebar

**Comment by javathehunt** [comment]
Thanks.
